website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -285,6 +285,53 @@ func main() {
 
 When the operator is not running in a cluster, the Manager will return an error on starting since it can't detect the operator's namespace in order to create the configmap for leader election. You can override this namespace by setting the Manager's `LeaderElectionNamespace` option.
 
+### Multiple architectures
```

Comment:
+1 for it is not in the Golang advanced topics. It seems valid for all languages. It would better fit as a section such as Scorecard (ihmo)

Reply:
I am happy to do that if there is consensus for it. @asmacdo @joelanford what is your take on this?